dtcInterfaceLib/DTC_Registers.h

```diff
@@ -978,6 +980,12 @@ class DTC_Registers : public CFOandDTC_Registers
 	void SetROCDCSResponseTimer(uint32_t timer);
 	RegisterFormatter FormatROCDCSResponseTimerPreset();
 
+	// Software DataRequests
+        void SetDataRequest(const DTC_EventWindowTag& ts);
```

Comment:
I think we should change the name pattern here from "DataRequest" to "SoftwareDataRequest" so it is clear what these functions do.

Reply:
done